DashAI/back/dataloaders/classes/dataset_dashai.py

```diff
@@ -0,0 +1,151 @@
+import json
+import os
+from typing import Dict, List
+
+from datasets import ClassLabel, Dataset, Value, load_from_disk
+
+
+class DatasetDashAI(Dataset):
```

Comment:
Same as before: I think it is more common to use the package name before the module name: `DashAIDataset`.
```suggestion
class DashAIDataset(Dataset):
```